cernopendata/views.py

```diff
@@ -126,7 +126,10 @@ def _build_agg_sub_agg_qs(subagg_agg_mapping, args):
     for arg, arg_values in args.items():
         if arg in aggs.keys():
             for arg_val in arg_values:
-                qs_values["f"].append(f"{arg}:{arg_val}")
+                if arg == "availability":
```

Comment:
Beware of the behaviour when people check/uncheck the box repetitively.  For example, try the following scenarios: _(quoting search result numbers for a partially loaded site)_

- search for something e.g. CMS, giving 70 results 
- click on the checkbox, the page is reloaded, the "cms" search string is gone, and you see "no results found!" message
- uncheck the box again, and you are back at a search not containing "cms" string, with 110 results

The same applies for facets:

- make an empty query
- click on Documentation > Guide facet, obtaining 22 results
- click on on-demand checkbox, page is reloaded, "No results found!" message displayed, and the facet selection is gone